charts/grafana/templates/_config.tpl

```diff
@@ -85,7 +85,7 @@ download_dashboards.sh: |
 {{- range $provider, $dashboards := .Values.dashboards }}
   {{- range $key, $value := $dashboards }}
     {{- if (or (hasKey $value "gnetId") (hasKey $value "url")) }}
-  curl -skf \
+  curl -skLf \
```

Comment:
I would prefer to make the configurable.

Reply:
Updated. This PR now allows users to override curl options for each individual dashboard.